common/app/templates/inlineJS/blocking/shouldEnhance.scala.js

```diff
@@ -23,21 +23,19 @@
     //
     // NOTE: this moves people into a category where they do not get important things such as commenting
     var isOlderIOSDevice = function () {
-        // This is NOT what we want to be doing long term. It is a stopgap measure only...
-        var olderIPadOnFront = @SplitOlderIPadsSwitch.isSwitchedOn && @item.isFront && window.devicePixelRatio === 1;
 
         if (navigator.platform === 'iPhone' || navigator.platform === 'iPad' || navigator.platform === 'iPod') {
             // I'm intentionally being a bit over zealous in the detection department here
-            return /.*(iPhone|iPad; CPU) OS ([3456])_\d+.*/.test(navigator.userAgent) || olderIPadOnFront;
+            return /.*(iPhone|iPad; CPU) OS ([3456])_\d+.*/.test(navigator.userAgent);
```

Comment:
So iPhones and iPads on iOS 3, 4, 5 or 6 will still not have _anything_ enhanced (is there anything _lower_ than iOS 3?). Any iPad running iOS 7 or above will have articles enhanced, while no iPads of any description will have fronts enhanced. iPhones running iOS 7 or above will be given the works for fronts and articles. I think that makes sense, but I just wanted to check these rules are definitely what we want.